### PR TITLE
Add changes due to new CLI

### LINKED ISSUE #59: Use new CLI
**Describe the bug**
https://github.com/ESMValGroup/ESMValCore/pull/605 changes the command line interface for the v2 release. It would be good to align the tutorial with the new interface, i.e. use for example:

```bash
esmvaltool recipes list
esmvaltool recipes show recipe_albedolandcover.yml
esmvaltool recipes get recipe_albedolandcover.yml

esmvaltool config get_config_user
esmvaltool config get_config_developer
esmvaltool run examples/recipe_python.yml
esmvaltool run --config_file config_user.yml recipe_python.yml
esmvaltool run recipe_albedolandcover.yml  --write_plots=False
```

Note that
- The config-user file is no longer strictly necessary. If it isn't specified, it will try to use the default
- Single config-user settings can also be passed to the command line
- You can use the `get` commend to copy recipes from the package to your local working directory

### PR DESCRIPTION
**Pull request by LisaBock**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [ ] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [ ] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [ ] Give this pull request a descriptive title.
- [ ] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [ ] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [ ] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #59

### CODE REVIEW

**Comment by SarahAlidoost** [comment]
@JaroCamphuijsen can you please merge this PR. We can solve the linter errors later.

File: _episodes/02-installation.md [hunk 1]
```diff
@@ -244,12 +244,12 @@ to display the command line help.
...
 > > In my case when I run
 > > ~~~
-> > esmvaltool --version
+> > esmvaltool version
 > > ~~~
 > > {: .bash}
 > > I get that my installed ESMValTool version is
 > > ~~~
-> > 2.0.0b9
+> > 2.0.0
```
Comment (JaroCamphuijsen) [inline _episodes/02-installation.md:252]: actually the realy output is `$ ESMValCore: 2.0.0
`

File: _episodes/03-configuration.md [hunk 2]
```diff
@@ -22,26 +22,17 @@ keypoints:
 The ``config-user.yml`` configuration file contains all the global level
 information needed by ESMValTool to run. This is an [YAML
 file](https://yaml.org/spec/1.2/spec.html). An example configuration file can be
-found in the root directory of the ESMValTool repository:
-[config-user-example.yml](https://github.com/ESMValGroup/ESMValTool/blob/master/config-user-example.yml).
-
-First, we make a working directory ``esmvaltool_tutorial``.
-In a new terminal, run:
+found in the ESMValCore repository:
+[config-user-example.yml](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/config-user.yml). You could download it by typing:
```
Comment (JaroCamphuijsen) [inline _episodes/03-configuration.md:26]:
```suggestion
[config-user-example.yml](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/config-user.yml).

You can generate a template configuration file by running:
```

File: _episodes/03-configuration.md [hunk 3]
```diff
@@ -67,6 +58,108 @@ This file contains the information for:
...
+# See config-developer.yml for an example. Set to [null] to use the default
+ config_developer_file: null
+# Get profiling information for diagnostics
+# Only available for Python diagnostics
+profile_diagnostic: false
+```
+
+## Destination directory
+
+The destination directory is the rootpath where ESMValTool will store its output,
```
Comment (JaroCamphuijsen) [inline _episodes/03-configuration.md:100]:
```suggestion
The destination directory is the rootpath where ESMValTool will store its output folders containing
```

File: _episodes/04-recipe.md [hunk 4]
```diff
@@ -94,7 +101,7 @@ the name: recipe_example.yml LINK
 >  1    # ESMValTool
 >  2    # recipe_example.yml
 >  3    ---
->  4    documentation:
+>  4    documentation
```
Comment (JaroCamphuijsen) [inline _episodes/04-recipe.md:104]: Why did you delete the colon? I think it should stay if I look at all the recipes...

File: _episodes/02-installation.md (hunk 1 above)
Comment (JaroCamphuijsen) [inline _episodes/02-installation.md:252]:
```suggestion
> > ESMValCore: 2.0.0
```

File: _episodes/04-recipe.md (hunk 4 above)
Comment (JaroCamphuijsen) [inline _episodes/04-recipe.md:104]:
```suggestion
>  4    documentation:
```
